cli/src/commands/config/mod.rs

```diff
@@ -81,19 +80,33 @@ impl ConfigLevelArgs {
         }
     }
 
-    fn edit_config_file(&self, command: &CommandHelper) -> Result<ConfigFile, CommandError> {
+    fn edit_config_file(
+        &self,
+        ui: &mut Ui,
```

Comment:
nit: `&mut` isn't required

Reply:
It's `&mut` all the way up, so I'd have to change a lot of other function signatures if you want me to remove it...

Reply:
Just change this function. `cmd_*()` should take `&mut Ui` because commands should be able to spawn pager.

FWIW, I've sent https://github.com/jj-vcs/jj/pull/5999 to simplify the prompt handling.